lib/genericCommand.js

```diff
@@ -115,11 +115,10 @@ module.exports = function(cmd) {
     if (typeof postCmd === 'function') {
       // Apply a useful scope in these functions
       var post = function(params, cb) {
-        postCmd.apply(cmd, [params, cb]);
+        postCmd.apply(cmd, [argv, params, cb]);
```

Comment:
This causes an error, try to run e.g.: `fhc app stage --app=cloud-app-id --env=environment-id --runtime=node4` You will receive:

fhc ERR! cb() never called!
fhc ERR!  
fhc Command executed with error
/home/trepel/tmp/camilla/fh-fhc/node_modules/async/lib/async.js:229
            callback(err, results);
            ^
```
TypeError: callback is not a function
    at /home/trepel/tmp/camilla/fh-fhc/node_modules/async/lib/async.js:229:13
    at /home/trepel/tmp/camilla/fh-fhc/node_modules/async/lib/async.js:110:21
    at /home/trepel/tmp/camilla/fh-fhc/node_modules/async/lib/async.js:24:16
    at /home/trepel/tmp/camilla/fh-fhc/node_modules/async/lib/async.js:226:17
    at /home/trepel/tmp/camilla/fh-fhc/lib/common.js:141:16
    at cb (/home/trepel/tmp/camilla/fh-fhc/lib/utils/request.js:113:9)
    at Request._callback (/home/trepel/tmp/camilla/fh-fhc/lib/utils/request.js:169:14)
    at Request.self.callback (/home/trepel/tmp/camilla/fh-fhc/node_modules/request/request.js:187:22)
    at emitTwo (events.js:87:13)
    at Request.emit (events.js:172:7)
```